### PR TITLE
JBIDE-27895: allow macos/safari to redirect to non-https urls

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

File: common/plugins/org.jboss.tools.common.oauth.ui/src/org/jboss/tools/common/oauth/internal/ui/OAuthBrowser.java [hunk 1]
```diff
@@ -16,16 +16,21 @@
...
 import org.eclipse.core.runtime.IProgressMonitor;
 import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Platform;
 import org.eclipse.core.runtime.Status;
 import org.eclipse.core.runtime.jobs.Job;
 import org.eclipse.swt.SWT;
 import org.eclipse.swt.browser.Browser;
 import org.eclipse.swt.events.DisposeEvent;
 import org.eclipse.swt.events.DisposeListener;
+import org.eclipse.swt.internal.cocoa.NSBundle;
```
Comment (jeffmaury) [inline common/plugins/org.jboss.tools.common.oauth.ui/src/org/jboss/tools/common/oauth/internal/ui/OAuthBrowser.java:26]: This is likely not to compile on non MacOS platforms

**Review by jeffmaury — CHANGES_REQUESTED** [review]

File: common/plugins/org.jboss.tools.common.oauth.ui/src/org/jboss/tools/common/oauth/internal/ui/OAuthBrowser.java (hunk 1 above)
Comment (adietish) [reply common/plugins/org.jboss.tools.common.oauth.ui/src/org/jboss/tools/common/oauth/internal/ui/OAuthBrowser.java:26]: @jeffmaury why not? All OS specific fragments are present at compile time, they're only filtered out on runtime. Btw. jenkins compiled it. What am I missing?

Comment (jeffmaury) [reply common/plugins/org.jboss.tools.common.oauth.ui/src/org/jboss/tools/common/oauth/internal/ui/OAuthBrowser.java:26]: Maybe Tycho is using all fragments but not Eclipse. It tried to load the PR in Eclipse and I got compilation failures

Comment (adietish) [reply common/plugins/org.jboss.tools.common.oauth.ui/src/org/jboss/tools/common/oauth/internal/ui/OAuthBrowser.java]: @jeffmaury turned to use reflection. Please test.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by adietish** [comment]
rebased (and pushed force) so that common is bumped and thus baseline checks should pass.

**Comment by adietish** [comment]
@jeffmaury had to rebase (for successful baseline check) and push force. Please approve again.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**